v19.2/create-table.md

```diff
@@ -264,74 +258,108 @@ In this example, we use `ON DELETE CASCADE` (i.e., when row referenced by a fore
-|            | ASC),                                                                    |
-|            |                                                                          |
-|            |     FAMILY "primary" (id, customer_id)                                   |
-|            |                                                                          |
-|            | )                                                                        |
-+------------+--------------------------------------------------------------------------+
+> SHOW CREATE vehicles;
+~~~
+
+~~~
+  table_name |                                          create_statement
++------------+-----------------------------------------------------------------------------------------------------+
+  vehicles   | CREATE TABLE vehicles (
+             |     id UUID NOT NULL DEFAULT gen_random_uuid(),
+             |     city STRING NOT NULL,
+             |     type STRING NULL,
+             |     owner_id UUID NULL,
+             |     creation_time TIMESTAMP NULL,
+             |     status STRING NULL,
+             |     current_location STRING NULL,
+             |     ext JSONB NULL,
+             |     CONSTRAINT "primary" PRIMARY KEY (city ASC, id ASC),
+             |     INDEX vehicles_auto_index_fk_city_ref_users (city ASC, owner_id ASC),
+             |     INVERTED INDEX ix_vehicle_ext (ext),
+             |     CONSTRAINT fk_owner_id_ref_users FOREIGN KEY (owner_id) REFERENCES users(id) ON DELETE CASCADE,
+             |     INDEX vehicles_auto_index_fk_owner_id_ref_users (owner_id ASC),
+             |     FAMILY "primary" (id, city, type, owner_id, creation_time, status, current_location, ext)
+             | )
 (1 row)
 ~~~
 
 {% include copy-clipboard.html %}
 ~~~ sql
-> INSERT INTO customers VALUES (1, 'Lauren');
+> INSERT INTO users (name, dl) VALUES ('Annika', 'ABC-123');
 ~~~
 
 {% include copy-clipboard.html %}
 ~~~ sql
-> INSERT INTO orders VALUES (1,1);
+> SELECT * FROM users;
+~~~
+
+~~~
+                   id                  | city |  name  | address | credit_card |   dl
++--------------------------------------+------+--------+---------+-------------+---------+
+  26da1fce-59e1-4290-a786-9068242dd195 | NULL | Annika | NULL    | NULL        | ABC-123
+(1 row)
+~~~
+
+{% include copy-clipboard.html %}
+~~~ sql
+> INSERT INTO vehicles (city, owner_id) VALUES ('seattle', '26da1fce-59e1-4290-a786-9068242dd195');
+~~~
+
+{% include copy-clipboard.html %}
+~~~ sql
+> SELECT * FROM vehicles;
+~~~
+
+~~~
+                   id                  |  city   | type |               owner_id               | creation_time | status | current_location | ext
++--------------------------------------+---------+------+--------------------------------------+---------------+--------+------------------+------+
+  fc6f7a8c-4ba9-42e1-9c37-7be3c906050c | seattle | NULL | 26da1fce-59e1-4290-a786-9068242dd195 | NULL          | NULL   | NULL             | NULL
```

Comment:
Interesting, so the DB schema doesn't auto-fill a creation time.  Huh.  Not material to this review, just a possible MovR code/DB update for the future, perhaps?  TBH it strikes me as odd that this field, status, and current location are allowed to be NULL.  Particularly location and status.

Reply:
I agree! There are a lot of gaps in the data generator like this... I am compiling a list of improvements to the dataset/generator. Setting the default values as non-null, generated values is definitely among on that list.